src/vs/workbench/contrib/chat/browser/actions/chatContext.ts

```diff
@@ -279,7 +279,7 @@ export class TerminalContext implements IChatContextValueItem {
 			return;
 		}
 
-		const command = terminal.capabilities.get(TerminalCapability.CommandDetection)?.commands.find(cmd => cmd.id === this._resource.query);
+		const command = terminal.capabilities.get(TerminalCapability.CommandDetection)?.commands.find(cmd => cmd.id === this._resource.query.replace('command=', ''));
```

Comment:
This isn't great either and will also fail if there is more than one query param. I think this is the right way to do it:

```ts
new URLSearchParams('commandId=abc&something=else').get('commandId')
```